Gems/Atom/Feature/Common/Assets/Materials/ReflectionProbe/ReflectionProbeVisualization.azsli

```diff
@@ -15,6 +15,8 @@
 
 #if MATERIALPIPELINE_SHADER_HAS_PIXEL_STAGE
     
+#define MATERIAL_HAS_UVS 0
```

Comment:
Where is the logic that uses `MATERIAL_HAS_UVS`?

Reply:
In non-public code 
The usage is essentially
MyCustomShader.azsli:
#if MATERIAL_HAS_UVS
    float2 uv = vertexData.uv;
#else
    float2 uv = float2(0, 0);
#endif